Tools/autotest/arduplane.py

```diff
@@ -1427,6 +1427,34 @@ def deadreckoning(self):
         self.deadreckoning_main()
         self.deadreckoning_main(disable_airspeed_sensor=True)
 
+    def rtl_climb_min(self):
+        self.wait_ready_to_arm()
+        rtl_climb_min = 100
+        self.set_parameter("RTL_CLIMB_MIN", rtl_climb_min)
+        takeoff_alt = 50
+        self.takeoff(takeoff_alt)
+        self.change_mode('CRUISE')
+        self.wait_distance_to_home(1000, 1500, timeout=60)
+        self.change_mode('RTL')
+        self.wait_altitude(90, 110, relative=True)
+        expected_alt = self.get_parameter("ALT_HOLD_RTL")/100.0
```

Comment:
maybe check half way if it is within a margin of mid-point of RTL alt and climb min + start alt